distribution/registry.go

```diff
@@ -126,17 +125,7 @@ func NewV2Repository(ctx context.Context, repoInfo *registry.RepositoryInfo, end
 }
 
 func digestFromManifest(m *schema1.SignedManifest, name reference.Named) (digest.Digest, int, error) {
```

Comment:
This helper doesn't seem very useful anymore. At least, there is no need to have `name` as input and `err` as output.